### ISSUE TITLE
Switcheroo not showing some Windows

### ISSUE THREAD

**Issue by benyaminl** [Wox-launcher/Wox.Plugin.Switcheroo#12]
Hello, I want to ask, some system windows on Windows doesn't showed on Switcheroo for Wox. Is there anyway possible fix for it? And sometimes I can't switch window need 2 until 3 times until it succeed, is there any possible way fix too? Thanks

**Comment by lances101** [comment]
@benyaminl As I've already stated in issue #11, the plugin is not responsible for finding the windows themselves, it uses [Switcheroo](http://www.switcheroo.io/). 

Sadly, Switcheroo's development has been abandoned (last update was 2 years ago) and the latest version does seem to have some issues. 

>  I can't switch window need 2 until 3 times until it succeed

Could you be more specific? Are you using Alt+Tab to switch?

**Comment by benyaminl** [comment]
@lances101 Yes, and it shows the wox with 'win' and the list of the windows running, and when I tried to change to other windows, it does nothing and dissapear, I need to restart wox or tried it 2-3 times, sometimes work, sometimes not. I just think, I write to fast and just press enter while changing the windows

**Comment by lances101** [comment]
There is no quick fix to that, but I can propose you a workaround:
Open Wox settings, go to Hotkeys and add a hotkey that queries `win`. Try using that instead of the Alt+Tab hook in the plugin itself. There are some focusing issues because of the way windows handles the Alt+Tab combination. I use the plugin this way myself and haven't had any issues.

**Comment by benyaminl** [comment]
@lances101 Okay, I'll, for now I tried to be slower, because now I depends on wox alot, since I remove windows explorer and use EmergeDesktop + Wox. Thanks for the replies
